packages/components/messages/src/info-message/info-message.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode } from 'react';
+import Text from '@commercetools-uikit/text';
+
+export type TIntlMessageDescriptor = {
+  id: string;
+  description?: string | object;
+  defaultMessage: string;
+  values?: Record<string, ReactNode>;
+};
+
+export type TInfoMessageProps = {
+  id?: string;
+  intlMessage?: TIntlMessageDescriptor;
+  children?: ReactNode;
+};
+
+const InfoMessage = (props: TInfoMessageProps) => (
```

Comment:
I had suggested to use the name `AdditionaInfoMessage` on purpose.

If we use only `InfoMessage`, we would think this is going to be using a blueish color as that's what we usually use the `info` word to associate that color in some other component.

Could you please rename this files/component to use the `AdditionalInfoMessage`? 🙏